Lib/statistics.py

```diff
@@ -695,6 +695,80 @@ def _ss(data, c=None):
     return (T, total)
 
 
+def _normal_dist_inv_cdf(p, mu, sigma):
```

Comment:
Minor nit:  Let's move this downward, just below the `## Normal Distribution #####################################################` section divider.